Dockerfile

```diff
@@ -18,6 +18,23 @@ RUN [ "$SKIP_TESTS" = 'false' ] && \
   bin/validate-values.sh && \
   bin/validate-templates.sh || true
 
+#-----------------------------
+FROM bats/bats:latest as test
```

Comment:
I prefer to add bats to our tools image and run on that, so we might catch missing deps while running tests.